SpatialGDK/Source/SpatialGDK/Private/EngineClasses/SpatialLoadBalanceEnforcer.cpp

```diff
@@ -117,7 +95,7 @@ void SpatialLoadBalanceEnforcer::ProcessQueuedAclAssignmentRequests()
 			// TODO(zoning): Not sure whether this should be possible or not. Remove if we don't see the warning again.
 			UE_LOG(LogSpatialLoadBalanceEnforcer, Warning, TEXT("(%s) Entity without AuthIntent component will not be processed. EntityId: %lld"), *WorkerId, Request.EntityId);
 			CompletedRequests.Add(Request.EntityId);
-			return;
+			return PendingRequests;
```

Comment:
Driveby - this should be continue - see https://github.com/spatialos/UnrealGDK/pull/1555

Reply:
Yes, this should be `continue`, so changing it to `return PendingRequests` should fix the issue

Reply:
But if you return you aren't continuing...

Reply:
All right :D I'll `continue`

Reply:
Addressed